functional_tests.py

```diff
@@ -125,7 +137,101 @@ def test_reset(self):
     def test_export(self):
         self.parser.export("test.csv")
         self.assertTrue(
-            os.path.exists(os.path.join("dist", "test.csv"))
+            os.path.exists(os.path.join("out", "test.csv"))
+        )
+
+
+class TestSeasonStatGrid(unittest.TestCase):
+
+    parser = leaders.SeasonStatGrid()
+
+    @classmethod
+    def setUpClass(cls):
+        cls.base_url = cls.parser.browser.current_url
+
+    @classmethod
+    def tearDownClass(cls):
+        for file in os.listdir("out"):
+            os.remove(os.path.join("out", file))
+        os.rmdir("out")
+        cls.parser.quit()
```

Comment:
`setUpClass` and `tearDownClass` are unnecessary if `TestSeasonStatGrid` has only one instance method. It is preferable that the tests are split up by method/functionality.